config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  namespace :api do 
+    namespace :v1 do
+      resources :greetings, only: [:index]
```

Comment:
- [ ] Your API is returning null, the message, therefore, is not being returned. Kindly check into that
![image](https://user-images.githubusercontent.com/62705947/204851393-6930b7bb-752a-4747-a122-659e22b7d79b.png)
![image](https://user-images.githubusercontent.com/62705947/204851462-9acae467-64ff-40f2-95c8-94cb18c2c610.png)

Reply:
hi man its null because you did not add any massages to Api end point on your end 
@GzimAsani 
here is an image prove that the project is working find 
![image](https://user-images.githubusercontent.com/95174521/204853493-4115a86b-7a71-4197-92ea-f08ff83a6490.png)

Reply:
all you need is to add some messages to the database you have created 
when you cloned the api from my repo 

#you can add to the database by using rails console 
just like that 

Greetings.create(message: 'hi')

Reply:
Hi @Alzubair98, I must have not add the greeting correctly. I apologize for the confusion caused